README.md

```diff
@@ -63,3 +66,9 @@ See LICENSE for more info.
 [freenode]: https://freenode.net/
 [irc]: irc://irc.freenode.net/bcoin
 [changelog]: https://github.com/bcoin-org/bcash/blob/master/CHANGELOG.md
+
+
+[coverage-status-img]: https://coveralls.io/repos/github/bcoin-org/bcoin/badge.svg?branch=master
```

Comment:
would these have to refer to `bcoin-org/bcash`? or is it all under the same umbrella for travis and coveralls?